src/common/git/getChangedFileLines.ts

```diff
@@ -3,21 +3,24 @@ import { exec } from "child_process";
 import { getGitHubEnvVariables, getGitLabEnvVariables } from "../../config";
 import { PlatformOptions } from "../types";
 export const getChangesFileLinesCommand = (
-  isCi: string,
+  isCi: string | undefined,
   fileName: string
 ): string => {
   if (isCi === PlatformOptions.GITHUB) {
     const { githubSha, baseSha } = getGitHubEnvVariables();
+
     return `git diff -U0 --diff-filter=AMRT ${baseSha} ${githubSha} ${fileName}`;
   } else if (isCi === PlatformOptions.GITLAB) {
     const { gitlabSha, mergeRequestBaseSha } = getGitLabEnvVariables();
+
     return `git diff -U0 --diff-filter=AMRT ${mergeRequestBaseSha} ${gitlabSha} ${fileName}`;
   }
+
   return `git diff -U0 --diff-filter=AMRT --cached ${fileName}`;
 };
 
 export const getChangedFileLines = async (
-  isCi: string,
+  isCi: string | undefined,
```

Comment:
[Note] It would be nice to rename this as the name implies it's boolean

Reply:
Changed to 'platform'

Reply:
Sorry I didn't make much sense. That would be a breaking change. Can you make a ticket to change this and we can do this in more of a controlled way as part of v2.

Reply:
Hmm- I don't think it is as the name of the argument is 'ci' and 'isCi' is only being used internally. I can change it back if you would like though

Reply:
Okay. whatever reads better then. I think `platformz is not the best as this variable is assigned to the value of the --ci flag. That is a large change in wording.

Reply:
I have reverted the commit where I changed this and we can discuss the right wording :)